### PR TITLE
Feature/#84 applications

### PR DESCRIPTION
**Pull request by lkolacz**

### CODE REVIEW

File: backend/unpp_api/apps/project/models.py [hunk 1]
```diff
@@ -89,6 +91,7 @@ class Application(TimeStampedModel):
     eoi = models.ForeignKey(EOI, related_name="applications", null=True, blank=True)
     submitter = models.ForeignKey('account.User', related_name="applications")
     cn = models.FileField()
+    cn_id = models.CharField(max_length=255, null=True, blank=True)
```
Comment (elyk) [inline backend/unpp_api/apps/project/models.py]: I don't think there is a need for this field. It's just database ID + CN.

File: backend/unpp_api/apps/project/serializers.py [hunk 2]
```diff
@@ -176,6 +178,36 @@ def create(self, validated_data):
...
 class PatchProjectSerializer(serializers.ModelSerializer):
 
+    specializations = SimpleSpecializationSerializer(many=True)
+    invited_partners = PartnerSerializer(many=True)
+    locations = PointSerializer(many=True)
+    assessments_criteria = AssessmentCriteriaSerializer(many=True)
+
     class Meta:
         model = EOI
         fields = "__all__"
```
Comment (elyk) [inline backend/unpp_api/apps/project/serializers.py]: We don't want all fields to be editable.  Only what was mentioned here: https://github.com/unicef/un-partner-portal/issues/83 (realize this PR is for #84 so not sure why it's here anyway). 

Would also call it `ProjectUpdateSerializer`

File: backend/unpp_api/apps/project/urls.py [hunk 3]
```diff
@@ -7,13 +7,17 @@
...
+    ApplicationAPIView,
+    ApplicationsListAPIView,
 )
 
 
 urlpatterns = [
     url(r'^(?P<pk>\d+)/$', EOIAPIView.as_view(), name="eoi-detail"),
     url(r'^(?P<pk>\d+)/partner-applications/$', ApplicationsPartnerAPIView.as_view(), name="partner-applications"),
     url(r'^(?P<pk>\d+)/agency-applications/$', ApplicationsAgencyAPIView.as_view(), name="agency-applications"),
+    url(r'^application/(?P<pk>\d+)$', ApplicationAPIView.as_view(), name="application"),
```
Comment (elyk) [inline backend/unpp_api/apps/project/urls.py]: Trailing slash

Comment (lkolacz) [reply backend/unpp_api/apps/project/urls.py]: solved on merge with developed

File: backend/unpp_api/apps/project/models.py (hunk 1 above)
Comment (lkolacz) [reply backend/unpp_api/apps/project/models.py]: cn_id removed

File: backend/unpp_api/apps/project/serializers.py (hunk 2 above)
Comment (lkolacz) [reply backend/unpp_api/apps/project/serializers.py]: done

**Comment by lkolacz** [comment]
@elyk improvements implemented - please take a look again, Thx!

**Pull request merged**